### DISCUSSION TITLE
Quick simple question

### DISCUSSION THREAD

**Discussion by msehabibur** [CederGroupHub/chgnet#211] [Q&A]
I'm curious about how CHGNET, which is trained on 0 K DFT data, can be used to perform dynamics simulations at finite temperatures, such as 1000 K. Since the model has not been exposed to any trajectories at elevated temperatures like 1000 K, how does it accurately capture finite-temperature behavior?

**Comment by janosh** [comment]
it may not. quite system dependent. there's certainly no guarantee but from past experience, 1000 K MD often works fine. the only way to know for sure is to try and then compare results to high quality reference data
